library.py

```diff
@@ -62,10 +67,11 @@ def check_out(filename, isbn, s_id):
 
     # Find a way to mark a book as checked out. Be sure to associate
     # the book with the student who borrowed it!
-
-
+    
+     f.__setitem__("filename",checked out) 
     # And again save the data here
-
+    with open("replayScript.json", "w") as jsonFile:
+    jsonFile.write(json.dumps(data))
```

Comment:
Another indentation error, and I think there's a simpler way to accomplish what you're trying to do here.